setup.py

```diff
@@ -85,23 +85,23 @@ def git_version():
 ]
 
 
-setup(name='adc',
+setup(name='adc-streaming',
       version=git_version(),
-      description='Genesis Platform client libraries',
+      description='Astronomy Data Commons streaming client libraries',
       long_description=open("README.md").read(),
       long_description_content_type="text/markdown",
-      url='https://github.com/astronomy-commons/genesis-client',
+      url='https://github.com/astronomy-commons/adc-streaming',
       classifiers=[
           "Programming Language :: Python :: 3",
           "License :: OSI Approved :: BSD License",
           "Development Status :: 3 - Alpha",
           "Operating System :: POSIX :: Linux",
           "Operating System :: MacOS :: MacOS X"
       ],
-      author='Mario Juric',
-      author_email='mjuric@astro.washington.edu',
-      license='MIT',
-      packages=['genesis'],
+      author='Astronomy Data Commons Team',
+      author_email='swnelson@uw.edu',
```

Comment:
This one too - let me know if it should be different.

Reply:
Looks good